client/src/components/pages/CreatePostPage/Event/index.js

```diff
@@ -0,0 +1,248 @@
+import React from "react";
+import { InputAntd, TextAreaAntd, DropDownAntd } from "components/utils";
+import "./style.css";
+import {
+  Form,
+  Input,
+  Tooltip,
+  Icon,
+  DatePicker,
+  AutoComplete,
+  InputNumber,
+  Upload,
+  Divider,
+  Card,
+  Button
+} from "antd";
+
+import { Button as Btn } from "components/utils";
+
+const InputGroup = Input.Group;
+const AutoCompleteOption = AutoComplete.Option;
+
+class EventForm extends React.Component {
+  state = {
+    confirmDirty: false,
+    autoCompleteResult: []
+  };
+
+  handleSubmit = e => {
+    e.preventDefault();
+    this.props.form.validateFieldsAndScroll((err, values) => {
+      if (!err) {
+        console.log("Received values of form: ", values);
+      }
+    });
+  };
+  handleWebsiteChange = value => {
+    let autoCompleteResult;
+    if (!value) {
+      autoCompleteResult = [];
+    } else {
+      autoCompleteResult = [".com", ".org", ".net"].map(
+        domain => `${value}${domain}`
+      );
+    }
+    this.setState({ autoCompleteResult });
+  };
+  render() {
+    const { autoCompleteResult } = this.state;
+    const { getFieldDecorator } = this.props.form;
+    const { eventTypeValues, eventTopicValues } = this.props;
+    const websiteOptions = autoCompleteResult.map(website => (
+      <AutoCompleteOption key={website}>{website}</AutoCompleteOption>
+    ));
+
+    return (
+      <Form className="main--eventForm" onSubmit={this.handleSubmit}>
```

Comment:
Class names with multiple words should be separated with dashes not written in camel case
```suggestion
      <Form className="main--event-form" onSubmit={this.handleSubmit}>
```